src/DacpacTool/DacpacTool.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;netcoreapp5.0</TargetFrameworks>
```

Comment:
net5.0 ??

Reply:
Yeah, that is what I had initially, but it conflicts with [this piece of code](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/192c4e992057ff1c4f4932d78dd0a50b522f95b0/src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets#L100). I guess we could fix that by adding a `Condition` for 5.0 and higher and a separate one for lower than 5.0. More ideally though we would have the TFM of the running MSBuild and use that, but I'm not sure if that's available.

Reply:
Fair enough

Reply:
@ErikEJ Looks like there is a `MicrosoftNETBuildTasksTFM` which is defined [here](https://github.com/dotnet/sdk/blob/8288f55cf35fb642bf2838bd11b6e17d884f019f/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets#L18). It should be available in every project, but I don't know if it is the right property to use. Looking at the history it seems to only have changed recently from `netcoreapp2.1` to `net5.0`, but it doesn't appear to have ever been `netcoreapp3.1`.

To be honest, I think it would be fine to keep it like it is currently. Unless the `netcoreapp5.0` TFM is going to be removed before .NET 5 ships. Do you happen to know if that's the case?

Reply:
I think it will continue to be available, at least for 5.0